release-notes.md

```diff
@@ -45,6 +45,16 @@ Android Places Monitor 1.0.0:
 
 For more information about the Places Monitor, [click here](https://github.com/Adobe-Marketing-Cloud/aep-sdks-documentation/tree/7ad91dd281c187baadf3698a3130f4ac0ef81fbe/places-extension-1/places-monitoring-extension/README.md).
 
+## May 15, 2019
+
+The following updates were made to the Adobe Target extension in Android version 1.1.0 and iOS version 2.1.0:
+
+ * Upgraded Target Delivery APIs to latest v1/delivery endpoint.
+ * Introduced new API `retrieveLocationContent` (API to retrieve content for multiple Target mbox locations at once without increasing reporting count for prefetch cases)
```

Comment:
Please reword so that it reads as follows:

"Introduced `retrieveLocationContent`, a new API that retrieves content for multiple Target mbox locations simultaneously without increasing the reporting count for prefetch cases."